src/main/resources/db/migration/V43_74__Rename_min_confidence_column_in_detectable_object_configuration.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS "detectable_object_configuration" RENAME COLUMN confidence TO min_confidence_for_delivery;
```

Comment:
No.

Do rename it. 

What I asked you to rename is the minimum_confidence provided only during JobAnnotationProcessed.